src/pages/Integrations/SplunkSetup/SplunkSetupForm.tsx

```diff
@@ -0,0 +1,161 @@
+import {
+    ActionGroup,
+    Button,
+    CodeBlock,
+    CodeBlockCode,
+    Form,
+    FormGroup,
+    Grid,
+    GridItem,
+    Popover,
+    ProgressStepProps,
+    TextInput } from '@patternfly/react-core';
+import CheckCircleIcon from '@patternfly/react-icons/dist/js/icons/check-circle-icon';
+import ExclamationCircleIcon from '@patternfly/react-icons/dist/js/icons/exclamation-circle-icon';
+import HelpIcon from '@patternfly/react-icons/dist/js/icons/help-icon';
+import InProgressIcon from '@patternfly/react-icons/dist/js/icons/in-progress-icon';
```

Comment:
Could you use the short import ?

```suggestion
import { CheckCircleIcon, ExclamationCircleIcon, HelpIcon, InProgressIcon } from '@patternfly/react-icons';
```